Automattic/pocket-casts-ios
# Transcripts: Style and scroll

Refs #1848 

This PR starts scrolling and style the transcript as the position in the stream moves.

## To test

- Start the app
- Ensure that `transscripts` and `newShowNotesEndpoint ` FF are enabled
- Open an episode with transcripts. Ex: any episode of Cautionary Tales
- Tap play
- Open the player full screen
- Open the transcripts tab
- Check if the transcript display and the text is highlights and scrolls as it's spoken.

## Checklist
